.github/workflows/release.yml

```
@@ -1,42 +1,33 @@
 name: Go package
 
 on:
-  push:
-    tags:
-      - "v*" # push events to tagged commits
-    branches:
-      - "**"
+  workflow_dispatch:
```

Why did you make this change? I think automatically pushing to docker.io on each merge to`defang` is a good idea.

Why shouldn't we do something like this?
```suggestion
  push:
    tags:
      - "v*" # push events to tagged commits
    branches:
      - "defang"
```

---

I am reluctant to have this auto push until we have a test in place.

---

A test of what?

---

end to end test. The fact that we have to handle differences between anthropic and gemini makes me think regressions will go undetected if we automate pushing.